nautobot/extras/tests/test_datasources.py

```diff
@@ -427,6 +482,19 @@ def populate_repo(path, url):
                     {"ntp-servers": ["172.16.10.22", "172.16.10.33"]},
                     config_context.data,
                 )
+                self.assertEqual(self.config_context_schema["_metadata"]["name"], config_context.schema.name)
```

Comment:
I think it would be clearer to instead `self.assertEqual(config_context_schema_record, config_context.schema)`?